CONTRIBUTING.md

```diff
@@ -1,31 +1,83 @@
+## Environment Setup
+
+SlashDeploy uses a `Gemfile` to document dependencies.
+
+The following external dependencies are also required:
+
+ * `Postgresql 9.6`
+ * `rbenv`
```

Comment:
rbenv is not required, just a working version of ruby

Reply:
I think I'll change this to Ruby 2.2.6 with a suggestion of rbenv.

```
cat .ruby-version
2.2.6
```